config/authors/aseemwangoo.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "aseemwangoo",
+  "bio": "",
```

Comment:
Founder of Flatteredwithflutter. YouTuber, blogger, event speaker and technical writer